costar_dataset/block_stacking_reader_torch.py

```diff
@@ -303,7 +304,8 @@ def encode_action_and_images(
 
     data_features_to_extract: A string identifier for the encoding to use for the actions and images.
         Options include: 'image_0_image_n_vec_xyz_aaxyz_nsc_15', 'image_0_image_n_vec_xyz_10',
-            'current_xyz_aaxyz_nsc_8', 'current_xyz_3', 'proposed_goal_xyz_aaxyz_nsc_8'.
+            'current_xyz_aaxyz_nsc_8', 'current_xyz_3', 'proposed_goal_xyz_aaxyz_nsc_8',
+            'image_n_vec_xyz_aaxyz_nsc_q_dq_gripper', 'image_m_image_n' .
```

Comment:
Please add enough comments explaining your new options so someone who never used this before can use it. I'll need to do the same for the other options myself.